### PR TITLE
js loops for in select nav challenge

### PR DESCRIPTION
**Pull request by CristobalCaroF**

### CODE REVIEW

**Review by versacrvm — COMMENTED** [review]
hola Cristobal, I was looking at your pr and noticed that you didn't set the option value in the first part of the exercise and didn't append the list item to the unordered list in the second part of the exercise. otherwise great work! :)

**Comment by CristobalCaroF** [comment]
Thanks for the comments! already added your comments, waiting for approval :)

**Review by versacrvm — APPROVED** [review]

**Pull request merged**
